### PR TITLE
simulator: make some parameters configurable

### PR DESCRIPTION
**Pull request by rleungx**
### What problem does this PR solve? 
The current simulator has some parameters hard-coded. It is not friendly enough.

### What is changed and how it works?
This PR is going to make some parameters of the simulator configurable and also renames some variables to make it more clear.

### Check List 

Tests 

 - Manual test

### CODE REVIEW

File: pkg/faketikv/config.go [hunk 1]
```diff
@@ -1,5 +1,66 @@
...
+const (
+	defaultSimTickInterval    = 100 * time.Millisecond
+	defaultNormTickInterval   = 1 * time.Second
+	defaultStoreCapacityGB    = 1024
+	defaultStoreAvailableGB   = 1024
+	defaultStoreIOMBPerSecond = 40
+	defaultStoreVersion       = "2.1.0"
+)
+
+// SimConfig is the simulator configurations.
```
Comment (disksing) [inline pkg/faketikv/config.go]: configurations -> configuration

File: pkg/faketikv/config.go [hunk 2]
```diff
@@ -1,5 +1,66 @@
...
+type SimConfig struct {
+	SimTickInterval  typeutil.Duration `toml:"sim-tick-interval"`
+	NormTickInterval typeutil.Duration `toml:"norm-tick-interval"`
+
+	StoreCapacityGB    uint64 `toml:"store-capacity"`
```
[the remark this answers is not in the document]
Comment (rleungx) [reply pkg/faketikv/config.go]: Since the definition of `store` is `uint64`, should I change them all?

[the remark this answers is not in the document]
Comment (disksing) [reply pkg/faketikv/config.go]: No. My point is here the config is by GB so we cannot express capacities like 0.5GB or 2.5GB if you use integers.

[the remark this answers is not in the document]
Comment (rleungx) [reply pkg/faketikv/config.go]: Oh, you're definitely right.

**Pull request merged**